### ISSUE TITLE
README: Add screenshots

### ISSUE THREAD

**Issue by luzpaz** [ehtec/ehtecoptics#2]
Screenshots would help users grok what this WB does

**Comment by ehtec** [comment]
You are right. Have not had a lot of time to continue work on this project recently. Will add some when I have time
